src/components/Blog.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component, Fragment} from 'react';
+import { Card, CardImg } from "react-bootstrap";
+import '../App.css';
+import { postsDb } from '../firebase';
+
+class BlogPage extends Component {
+
+        state = {
+            postsList: []
+        }
+
+    componentDidMount(){
+       postsDb.ref('posts').on("value", snapshot => {
+            let postlist = [];
+            snapshot.forEach(snap => {
+                postlist.push(snap.val());
+               
+            })
+            this.setState({postsList: postlist})
+        })
+    }
+    componentWillUnmount(){
+        postsDb.ref('posts').off()
+    }
+
+render(){
+   
+    return(
+    <Fragment>
+        <div className="container blog">
+            <h1 className="text-center">My Blogs</h1>
+            <div className="d-flex flex-wrap">
+            {this.state.postsList.map(post => (
+                <Card className="article-card" key={post}>  
+                    <CardImg className="card-image-top" 
+                        width="100%"
+                        src={post.url}
+                        alt="blogPicture"
+                    />
+                <Card.Body className="card-body">
+                    <Card.Title className="card-title" key={post.title}>
```

Comment:
I could be wrong, but I don't think you need `key` on these. You usually just need `key` on the top-level component in the `.map()`. So I think you just need `key` on `Card` but nothing under `Card`.